packages/resource-deployment/runtime-config/runtime-config.ci.json

```diff
@@ -0,0 +1,3 @@
+{
+    "profileName": "ci"
```

Comment:
define configuration that is similar to this structure - https://github.com/microsoft/accessibility-insights-service/blob/users/devaradhanm/config3/packages/common/src/configuration/service-configuration.ts

Else, this will break

Reply:
Sure. I created this file only for testing purpose.